### DISCUSSION TITLE
How to use multiple sequential keys for a keybind

### DISCUSSION THREAD

**Discussion by GV14982** [wezterm/wezterm#4794] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

Hi, I am wondering if it is possible to have multiple sequential key presses for a keybind. For instance I want to use `cp` for closing a pane and `ct` for closing a tab.

**Comment by wez** [comment, marked as answer]
You may be able to this with https://wezfurlong.org/wezterm/config/key-tables.html

**Reply by GV14982** [reply]
Awesome, sorry I missed this in the docs. Thanks so much!
